docs_app/content/navigation.json

```diff
@@ -24,7 +24,13 @@
 		{
 			"url": "guide/overview",
 			"title": "Overview",
-			"tooltip": "RxJS Overview"
+			"tooltip": "RxJS Overview",
+			"children": [
+				{
+					"url": "guide/subject",
+					"title": "Subjects in RxJS"
```

Comment:
'Subjects' would be totally fine for the title. I think it is totally clear that it is about rxjs :D